main.go

```diff
@@ -3,18 +3,27 @@ package main
 import (
 	"flag"
 	"os"
+	"time"
 
 	"github.com/Azure/kubelogin/pkg/cmd"
 	"github.com/spf13/pflag"
 	klog "k8s.io/klog/v2"
 )
 
+var timeoutDuration time.Duration
+
 func main() {
 	klog.InitFlags(nil)
+
+	flag.DurationVar(&timeoutDuration, "timeout", 10*time.Second, "Timeout duration for Azure CLI token requests")
+
 	pflag.CommandLine.AddGoFlag(flag.CommandLine.Lookup("v"))
 	pflag.CommandLine.AddGoFlag(flag.CommandLine.Lookup("logtostderr"))
+	pflag.CommandLine.AddGoFlag(flag.CommandLine.Lookup("timeout"))
```

Comment:
can we add this flag to just get-token subcommand?

Reply:
also, it's discouraged to mix pflag with flag. it should be straightforward to add to https://github.com/Azure/kubelogin/blob/master/pkg/token/options.go#L96

Reply:
fixed